test/e2e/run-canary-test.sh

```diff
@@ -62,21 +62,16 @@ eksctl create iamserviceaccount \
 --override-existing-serviceaccounts \
 --approve || true
 
-echo "Update helm repo eks"
-helm repo add eks https://aws.github.io/eks-charts
```

Comment:
This script is not linked to the e2e tests. This is part of our internal canary where we test the release version.